src/Framework/BaseWebApplication.php

```diff
@@ -347,6 +347,11 @@ protected function configureTemplateEngine()
         if (isset($this['parameters']['userbase_url'])) {
             $this['twig']->addGlobal('userbase_url', $this['parameters']['userbase_url']);
         }
+        
+        $this['twig']->getExtension('core')->setDateFormat('d/m/Y', '%d days');
+        if(isset($this['parameters']['twig_date_format'])) {
+            $this['twig']->getExtension('core')->setDateFormat($this['parameters']['twig_date_format'], '%d days');
+        }
```

Comment:
@bvklim @joostfaassen we already have parameters to specify the date/time formats. maybe it's good to reuse that?


<pre>
format:

        date: Y-m-d

        time: H:i

        datetime: "Y-m-d H:i"
</pre>